pkg/parser/epub/positions_service.go

```diff
@@ -1,5 +1,195 @@
 package epub
 
-type PositionsService struct {
-	// TODO
+import (
+	"math"
+
+	"github.com/readium/go-toolkit/pkg/fetcher"
+	"github.com/readium/go-toolkit/pkg/manifest"
+	"github.com/readium/go-toolkit/pkg/service"
+)
+
+/**
+ * Positions Service for an EPUB from its [readingOrder] and [fetcher].
+ *
+ * The [presentation] is used to apply different calculation strategy if the resource has a
+ * reflowable or fixed layout.
+ *
+ * https://github.com/readium/architecture/blob/master/models/locators/best-practices/format.md#epub
+ * https://github.com/readium/architecture/issues/101
+ */
+type EPUBPositionsService struct {
+	readingOrder       []manifest.Link
+	presentation       *manifest.Presentation
+	fetcher            fetcher.Fetcher
+	reflowableStrategy ReflowableStrategy
+	positions          [][]manifest.Locator
+}
+
+func (s *EPUBPositionsService) Close() {}
+
+func (s *EPUBPositionsService) Links() []manifest.Link {
+	return []manifest.Link{service.PositionsLink}
 }
+
+func (s *EPUBPositionsService) Get(link manifest.Link) (fetcher.Resource, bool) {
+	return service.GetForPositionsService(s, link)
+}
+
+// Positions implements PositionsService
```

Comment:
Then, to disambiguate use the qualified name in the doc comments.

```suggestion
// Positions implements pub.PositionsService
```